lxd/certificates.go

```diff
@@ -143,15 +143,23 @@ func certificatesGet(d *Daemon, r *http.Request) response.Response {
 		var err error
 		err = d.cluster.Transaction(func(tx *db.ClusterTx) error {
 			baseCerts, err = tx.GetCertificates(db.CertificateFilter{})
-			return err
+			if err != nil {
+				return err
+			}
+
+			for _, baseCert := range baseCerts {
```

Comment:
The `body := []string{}` var further down could get similar treatment too.

Reply:
I've applied this to the `certResponses` slice, but the `body` slice isn't as simple since it's a nested loop. Getting the right length would require looping over the 2 loops first to get the total length.